lunatrace/bsl/hasura/metadata/databases/lunatrace/tables/public_builds.yaml

```diff
@@ -1,174 +1,174 @@
 table:
```

Comment:
When applying metadata I am getting: 
```╰─$ hasura metadata apply 
FATA[0001] error applying metadata 
{
  "path": "$.args.metadata.remote_schemas[0].permissions[0].definition.schema",
  "error": "parsing the schema document: \"type definition: string\"",
  "code": "parse-failed"
} 
```
Does it work for you if you do hasura metadata apply from this branch?